apps/emqx_ft/test/emqx_ft_SUITE.erl

```diff
@@ -361,13 +403,134 @@ t_assemble_crash(Config) ->
     C = ?config(client, Config),
 
     meck:new(emqx_ft_storage_fs),
-    meck:expect(emqx_ft_storage_fs, assemble, fun(_, _) -> meck:exception(error, oops) end),
+    meck:expect(emqx_ft_storage_fs, assemble, fun(_, _, _) -> meck:exception(error, oops) end),
 
     ?assertRCName(
         unspecified_error,
         emqtt:publish(C, <<"$file/someid/fin">>, <<>>, 1)
     ).
 
+t_unreliable_migrating_client(Config) ->
+    NodeSelf = node(),
+    [Node1, Node2] = ?config(cluster_nodes, Config),
+
+    ClientId = ?config(clientid, Config),
+    FileId = emqx_guid:to_hexstr(emqx_guid:gen()),
+    Filename = "migratory-birds-in-southern-hemisphere-2013.pdf",
+    Filesize = 1000,
+    Gen = emqx_ft_content_gen:new({{ClientId, FileId}, Filesize}, 16),
+    Payload = iolist_to_binary(emqx_ft_content_gen:consume(Gen, fun({Chunk, _, _}) -> Chunk end)),
+    Meta = meta(Filename, Payload),
+
+    Context = #{
+        clientid => ClientId,
+        fileid => FileId,
+        filesize => Filesize,
+        payload => Payload
+    },
+    Commands = [
```

Comment:
The scenario is quite clear in detail, but the full list of purposes is a bit unclear.
M.b. add several comments to clarify what we are modelling and what the desired assertions mean?

Reply:
Attempted to describe better in a0ff75ddd04986b9400e9c447a5f6eba17e7519d.